ios/Integreat/AppDelegate.m

```diff
@@ -46,4 +48,39 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 #endif
 }
 
+- (void)clearStorage
```

Comment:
Minor: Would be cool to have this code as swift code. Don't really understand Objective C :D